canary_config.example.py

```diff
@@ -11,7 +11,12 @@
 SW_PWM_PIN = "20"
 SW_INTERRUPT_PIN = "21"
 
-creds = Credentials(
-    type="robot-location-secret",
-    payload="put-the-secret-here")
+creds = RobotClient.Options.with_api_key(
+    api_key="secret-api-key",
+    api_key_id="put-the-secret-id-here")
```

Comment:
This change is unrelated to the rest of the PR, but the way to get credentials to connect to a robot have changed since this was originally written.